src/main/window-manager.ts

```diff
@@ -89,6 +90,9 @@ export class WindowManager {
         frame: false,
         resizable: false,
         show: false,
+        webPreferences: {
+          nodeIntegration: true
```

Comment:
Integration tests requires nodeIntegration, and it determines it based on window nodeIntegration option, so we need to enable it also for splashWindow.